### ISSUE TITLE
"Lookup requires cgo" error when testing the README specs example

### ISSUE THREAD

**Issue by jordiclariana** [cloudflare/certmgr#30]
When following the README.md and trying it with the [specs](
https://github.com/cloudflare/certmgr#certificate-specs) provided it fails with this error:

```
# ./certmgr-linux-amd64-v1.4.2 check
2017/11/09 11:54:41 [INFO] certmgr: loading from config file /etc/certmgr/certmgr.yaml
2017/11/09 11:54:41 [INFO] manager: loading certificates from/etc/certmgr.d
2017/11/09 11:54:41 [INFO] manager: loading spec from /etc/certmgr.d/test.json
Failed: user: Lookup requires cgo
```

Steps to reproduce on a Ubuntu Server 16.04:

* Download latest certmgt version for linux (amd64) from `releases`: https://github.com/cloudflare/certmgr/releases/download/v1.4.2/certmgr-linux-arm64-v1.4.2.tar.gz
* Run genconfig: `./certmgr-linux-amd64-v1.4.2 genconfig`:
```
-----
dir: /etc/certmgr.d
svcmgr: systemd
before: 72h
interval: 1h
metrics_address: localhost
metrics_port: "8080"

-----
certmgr: creating certificate spec directory /etc/certmgr.d
```
* Create `/etc/certmgr.d/test.json` and paste _example certificate spec_ from the README.md
* Run `./certmgr-linux-amd64-v1.4.2 check` and get the error:
```
2017/11/09 12:00:25 [INFO] certmgr: loading from config file /etc/certmgr/certmgr.yaml
2017/11/09 12:00:25 [INFO] manager: loading certificates from/etc/certmgr.d
2017/11/09 12:00:25 [INFO] manager: loading spec from /etc/certmgr.d/test.json
Failed: user: Lookup requires cgo
```

If I remove any reference to `"owner" : "www-data"` and `"group" : "www-data"` from specs file, it works.

**Comment by kisom** [comment]
I think this is due to a [bug in pre-Go 1.9](https://github.com/golang/go/issues/19395). I will try to get this rebuilt with a newer Go.

**Comment by kisom** [comment]
This should be fixed in [v1.4.3](https://github.com/cloudflare/certmgr/releases/tag/v1.4.3).

**Comment by jordiclariana** [comment]
Awesome, thanks!
